flake.nix

```
@@ -12,6 +12,20 @@
       lib = import ./lib;
 
       forAllSystems = lib.genAttrs lib.systems.flakeExposed;
+
+      pkgsMemoizer = { pkgs, inputsSet, ... }:
+        let system = lib.systems.toLosslessStringMaybe inputsSet.hostPlatform.value.system;
```

```suggestion
        let system = lib.systems.toLosslessStringMaybe inputsSet.hostPlatform.value.system;
```

Calling this on `system` sounds wrong :laughing: